ajax/libs/inheritance.js/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "inheritance.js",
+  "npmName": "inheritance-js",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "*.js"
+      ]
+    }
+  ],
+  "filename": "inheritance.min.js",
+  "version": "0.1.2",
+  "description": "A simple and small implementation of object inheritance using pure JavaScript.",
+  "homepage": "https://github.com/bsara/inheritance.js#readme",

```

Comment:
We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo. Let's remove it. Thanks.